Parse/src/main/java/com/parse/ParsePushBroadcastReceiver.java

```diff
@@ -261,6 +264,27 @@ protected void onPushOpen(Context context, Intent intent) {
   }
 
   /**
+   * Retrieves the channel to be used in a {@link Notification} if API >= 26, if not null. The default returns a new channel
+   * with id "parse_push", name "Push notifications" and default importance.
+   *
+   * @param context
+   *      The {@code Context} in which the receiver is running.
+   * @param intent
+   *      An {@code Intent} containing the channel and data of the current push notification.
+   * @return
+   *      The notification channel
+   */
+  @TargetApi(Build.VERSION_CODES.O)
+  protected NotificationChannel getNotificationChannel(Context context, Intent intent) {
+    NotificationManager nm = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+    NotificationChannel notificationChannel = new NotificationChannel("parse_push", "Push notifications", NotificationManager.IMPORTANCE_DEFAULT);
+
+    // Android doesn't create a new channel if the properties of the channel hasn't changed
+    nm.createNotificationChannel(notificationChannel);
```

Comment:
Could you move the channel creation outside, maybe in `getNotification()`, right after we call `getNotificationChannel()` ? If someone overrides this method, the creation won't happen